crates/octez/src/async_node.rs

```diff
@@ -36,12 +36,13 @@ impl AsyncOctezNode {
                 network,
                 "--data-dir",
                 self.octez_node_dir.to_str().expect("Invalid path"),
+                // the node executable does not accept the scheme part
                 "--rpc-addr",
-                rpc_endpoint,
+                &rpc_endpoint.to_string().split("://").last().unwrap(),
```

Comment:
we can make a method that will create a string only for the host and port? would prob be useful

Reply:
Sure. Naming is the main problem. If something like `to_string_without_scheme` sounds okay, I can implement that.

Reply:
perhaps authority?
https://developer.mozilla.org/en-US/docs/Web/URI/Authority

Reply:
That's not necessarily enough since one can set up a node behind something such that it's accessible at a certain path

Reply:
right, could go without_scheme then!

Reply:
Sorry I was just overthinking. Authority should be enough.